Add TWIR 07-06-2024 (#297)
rails/website: _posts/2024-06-07-this-week-in-rails.markdown

---
layout: post
title: "Global strict loading mode setting, route draw deferring and more"
categories: news
author: Wojtek
og_image: assets/images/this-week-in-rails.png
published: true
date: 2024-06-07
---

Hi, [Wojtek](https://x.com/morgoth85) here. Let's explore this week's changes in the Rails codebase.

[Rails security releases](https://rubyonrails.org/2024/6/4/Rails-Versions-6-1-7-8-7-0-8-4-and-7-1-3-4-have-been-released)  
Rails Versions 6.1.7.8, 7.0.8.4, 7.1.3.4, and 7.2.0.beta2 have been released with a security issue fix.

[The CFP for RubyConf is open](https://sessionize.com/rubyconf-2024)  

> There will be a *Ruby Web Framework* theme and they are open to Rails related submissions.

Reviewer: ```suggestion
There will be a *Ruby Web Framework* theme, so get your Rails related submissions ready!
```
